src/screens/profile/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { MainButton, Text } from 'src/components';
+import { AppColor, ButtonMode } from 'src/common/enums';
+import { BUTTON_PROFILE_NAVBAR_WIDTH } from 'src/common/constants';
+import styles from './styles';
+
+type State = Array<{ id: number; text: string }>;
+
+interface Props {
+  active: number;
+  handleClick: (a: number) => void;
+}
+
+const Navbar = ({ active, handleClick }: Props) => {
+  const scrollRef = useRef<ScrollView>(null);
+
+  useEffect(() => {
+    const scrollToPosition = active * BUTTON_PROFILE_NAVBAR_WIDTH;
```

Comment:
We should use nouns for variables and verbs for functions
```suggestion
    const xOffset = active * BUTTON_PROFILE_NAVBAR_WIDTH;
```